person.rb

```diff
@@ -22,6 +23,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Your method of implementing this method is wrong. Since this is a person class it should take in book and date as arguments and create a rental with it.

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```